Dockerfile

```diff
@@ -132,6 +132,7 @@ ENV ANCHORE_CONFIG_DIR=/config \
     ANCHORE_AUTH_ENABLE_HASHED_PASSWORDS=false \
     AUTHLIB_INSECURE_TRANSPORT=true \
     ANCHORE_MAX_COMPRESSED_IMAGE_SIZE_MB=-1 \
+    ANCHORE_GRYPE_DB_URL="https://toolbox-data.anchore.io/grype/databases/listing.json" \
```

Comment:
(nit) add new variables at the end of the current list, makes it a tiny bit easy for folks to find new additions

Reply:
yeah got reordered during merge conflict resolution

Reply:
fixed